dotcom-rendering/src/web/components/ElementContainer.tsx

```diff
@@ -55,6 +59,7 @@ type Props = {
 	sectionId?: string;
 	showSideBorders?: boolean;
 	showTopBorder?: boolean;
+	showBottomBorder?: boolean;
```

Comment:
Is there a way to ensure this prop doesn't remain here? Maybe we can create the PR to revert the test code now and keep it as a draft while the test runs?

The reason a bottom border is a problem is 1. because we want this component to follow (and perhaps eventually be replaced with) the `Container` from Source and also 2. By allowing a top + a bottom border you create the conditions for double borders - which are a chronic problem on the old site. By only allowing one border we help prevent this.

Reply:
I sort of agonised over adding this prop, I really didn't want to touch `ElementContainer` but it was the only way I could think of to achieve visual consistency with frontend - the top slot looked *very* weird without the bottom border when scrolling past the nav bar! 
I knew there must be a reason why that prop wasn't there in the first place, I just didn't know what it was, so thanks for explaining! 
Good shout about the revert PR, I'll work on that now. I have also carded this up in our comm dev Trello to make sure the prop is removed when the A/B test is over.